README.md

```diff
@@ -1,2 +1,49 @@
-# first-capstone-project
-this is the first capstone project for the first module in microverse
+
+# Project Name
+First capstone project in microverse 
+# Project Link
+https://alzubair98.github.io/first-capstone-project/
+
+> Description the project.
+this is the first capstone project where i will build a university portofilo
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://alzubair98.github.io/first-capstone-project/)
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] Kindly edit the info of the README file `Project Name`, `Description`, `Build with` and `steps to get a local copy` if exist.
- [ ] [OPTIONAL] Please add an image for the website in your README in order to make your README more professional.

Reply:
done